src/Backend/ArrayCache.php

```diff
@@ -32,7 +32,7 @@ public function doSave($id, $data, $lifeTime = 0)
     public function doDelete($id)
     {
         if (!$this->doContains($id)) {
-            return false;
+            return true;
```

Comment:
@mattab  I have just read about this in the changelog. Have you tested how this behaves with other backends? Otherwise there might be bugs when they behave differently. Users can configure to use redis cache backend, file cache etc